tests/testthat/test-plot-with-ospsuite-plots.R

```diff
@@ -182,7 +209,7 @@ test_that("Function merges dimensions correctly", {
                           molWeight = 2
                           )
 
-  result <- .convertUnitsForPlot(validData, 2)
+  .convertUnitsForPlot(validData, 2)
```

Comment:
@rengelke should be changed back to result <- .convertUnitsForPlot(validData, 2) as result is needed for the following lines